src/site/filters/liquid.js

```diff
@@ -249,12 +244,36 @@ module.exports = function registerFilters() {
       const bTime = Math.floor(
         new Date(b.fieldDate.startDate).getTime() / 1000,
       );
-      const sorter = bTime - aTime;
+      // Sort order needs to be oldest first.
+      const sorter = aTime - bTime;
       return sorter;
     });
     return sorted;
   };
 
+  liquid.filters.pastPresent = (items, url) => {
+    const pastEventTeasers = [];
+    const currentEventTeasers = [];
+    // separate current events from past events;
+
+    _.forEach(items, value => {
+      const startDate = value.fieldDate.startDate;
+      const isPast = moment().diff(startDate, 'days');
+      if (isPast >= 1) {
+        pastEventTeasers.push(value);
+      } else {
+        currentEventTeasers.push(value);
+      }
+    });
+
+    const pathArray = url.split('/');
+    if (pathArray.includes('past-events')) {
+      return pastEventTeasers;
+    }
+
+    return currentEventTeasers;
+  };
```

Comment:
I had originally written up a comment on this but I guess I didn't save it before I submitted the previous review.


The goal seems to be filtering out past or present events depending on the url, so an actual feels more appropriate than a for loop where you push things onto a new array.
```suggestion
liquid.filters.pastPresent = (items, url) => {                                              
  const isPast = value => moment.diff(value.fieldDate.startDate, 'days') >= 1;        
  const isCurrent = value => !isPast(value);       

  const filter = url.split('/').includes('past-events') ? isPast : isCurrent;
  return items.filter(filter);                                                                  
 };
```

Reply:
:+1: